src/index.js

```diff
@@ -6,15 +6,23 @@ exports.register = function (server, options, next) {
   var client = new raven.Client(options.dsn, options.client);
   server.expose('client', client);
   server.on('request-error', function (request, err) {
+    var baseUrl = request.info.uri;
```

Comment:
it's not available in some cases, but that's what L10-13 is for oO
info.uri is configurable when setting up hapi, and falls back to browser sent stuff.